doc/fuzzing.md

```diff
@@ -28,9 +32,13 @@ export AFL_HARDEN=1
 cd src/
 make test/test_bitcoin_fuzzy
```

Comment:
why not adding something about speeding the process using `-j #NUM_CORES`?

Reply:
Yes, good point, I can do that. I just implicitly assumed people know about it, but maybe they'll forget if there's a command line like this presented.